package.json

```diff
@@ -76,7 +68,6 @@
     "prepack": "check-for-leaks",
     "prepush": "check-for-leaks",
     "prepare-release": "node ./script/prepare-release.js",
-    "prerelease": "python ./script/bootstrap.py -v --dev && npm run build",
```

Comment:
We may need this for the release process, though given where we are moving with automated releases it may not necessary.